compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -122,33 +122,24 @@ public function format(Diagnostic d) returns string[] {
     line += ": error: " + d.message;
     lines.push(line);
     Range|Position range = loc.range;
-    lines.push(range is Position ? loc.file.lineContent(range) : loc.file.lineContent(range.startPos));
+    [string, string, string] sourceLine = range is Position ? loc.file.lineContent(range) : loc.file.lineContent(range.startPos);
```

Comment:
Why aren't you just passing loc.range to lineContent? They are both `Range|Position`.

Reply:
fixed